validator/docs/schemas.md

```diff
@@ -0,0 +1,195 @@
+# Configuration Validator
+
+This module contains the logic needed to validate deployment files with a defined schema by Nestor.
+
+The deployment files define applications or projects.
+
+## Table of Contents
+1. [Application Configuration](#applicationConfiguration)
+2. [Kubernetes Configuration](#kubernetesConfiguration)
+
+<a name="applicationConfiguration"></a>
+## Application Configuration
+
+In each file you will define the application that you want to deploy, few fields are required to setup a new application. There are many supported values which are optional that could be useful for your deployment. You can read more about them in the following sections and the schemas definition.
+
+<a name="applicationConfiguration"></a>
+#### Application Example 
+This is how a basic configuration for an application looks like, this example contains the minimum required fields.
+
+```yaml
+app: my-application         # Application's name
+is_enabled: false           # Is the application deployment enabled
+git: git@microservice_repo  # Git repository URL
+variables:                  # Environment variables
+  app:                      
+    APP_VAR1: application_variable_1
+  ope:
+    OPE_VAR1: ope_variable_1
+```
+
+To further customize the deployment of your application, you can add the following sections to the configuration files.
+
+### Processes
+
+A project can contain multiple processes such as workers, web processes and more.
+The structure to define a process is the following one:
+
+```yaml
+# Defining a simple process
+processes:
+  - name: process-name        # Process name - required - maxLength: 58
+    is_cronjob: false         # Defining a simple process
+    start_command: npm start  # Command executed when the pod is started
+# Defining a scheduled process
+  - name: cronjob-name
+  is_cronjob: true
+  start_command: npm run worker
+```
+
+### Crons
+Crons are tasks that run at a specific time or interval.
+You can use them to automate tasks, commonly as: reporting, backups, maintenance, emails and more.
+
+You can define the crons configuration with the following structure.
+
+```yaml
+crons:
+  my-cronjob: # Cronjob to define, it must be the same name as the cronjob defined in processes
+    concurrency_policy: 'Forbid' # Allow, Forbid or Replace concurrency of cronjobs, STRING - Required. [Kubernetes documentation](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy)
```

Comment:
This line is really long, put the link after the block as a note to keep the comments simple.
And align the other comments together 🙂